v3/src/components/inspector-panel.scss

```
@@ -50,3 +50,78 @@
     }
   }
 }
+
+.codap-inspector-palette {
+  background-color: #EEE;
+  border: 0px #444 solid;
+  position: absolute;
+  left: 5px !important;
+  inset: 0px;
```

If you reverse the order you don't need the `!important`:
```suggestion
  inset: 0;
  left: 5px;
```